Backpack/src/main/java/net/skyscanner/backpack/badge/BpkBadge.kt

```diff
@@ -121,7 +121,7 @@ open class BpkBadge @JvmOverloads constructor(
      * @property type
      * Type of badge. Default Type.Success
      */
-    var type: Type = Type.Success
+    var type: Type = Type.Normal
```

Comment:
i think you also need to update this line:
`type = Type.fromId(a.getInt(R.styleable.BpkBadge_badgeType, 1))`
the 1 defines the default type as read in xml and matches the id defined in the enum

Reply:
Thanks Maria! I think this should be fixed now.